native/src/__mocks__/@react-native-community/async-storage.js

```diff
@@ -1,5 +0,0 @@
-// @flow
```

Comment:
Because I just added it directly to the jest setup as there is never a case where it makes sense to have the unmocked async storage in tests. It was also just a reference to the actual mock.